src/devtools/client/debugger/src/components/SecondaryPanes/Breakpoints/PanelStatus.tsx

```diff
@@ -29,15 +30,19 @@ export function PanelStatus({
   if (!isIndexed || !analysisPoints) {
     status = "Loading";
   } else if (analysisPoints.error) {
-    status = analysisPoints.error === AnalysisError.TooManyPoints ? "10k+ hits" : "Error";
-  } else if (analysisPoints.data.length == 0) {
+    status =
+      (analysisPoints.error as AnalysisError) === AnalysisError.TooManyPointsToFind ||
```

Comment:
Similarly, typescript thinks this things type is `AnalysisError.TooManyPointsToFind | AnalysisError.Unknown`, and so it thinks there is no overlap here. I'm sure it's doing some type inference somewhere, but it's unclear where?